src/m3/ecore/importer.ts

```diff
@@ -0,0 +1,119 @@
+import {
```

Comment:
Would it make any sense to use some of the EMF libraries for JS/TS? 
For example: https://emfjson.github.io/ecore.js/ or https://github.com/crossecore/ecore-typescript?